metron-platform/metron-common/src/main/java/org/apache/metron/common/stellar/StellarCompiler.java

```diff
@@ -263,7 +263,56 @@ public void exitVariable(StellarParser.VariableContext ctx) {
 
   @Override
   public void exitStringLiteral(StellarParser.StringLiteralContext ctx) {
-    expression.tokenDeque.push(new Token<>(ctx.getText().substring(1, ctx.getText().length() - 1), String.class));
+    String rawToken = ctx.getText();
+    String literal = unquoteLiteral(rawToken);
+
+    expression.tokenDeque.push(new Token<>(literal, String.class));
+  }
+
```

Comment:
Dude!  Great catch!  We're actually closer to JSON escaping because our strings can start with either `'` or `"`, so I used that escaping strategy and it worked just fine.  Thanks!